main.html

```diff
@@ -4,12 +4,11 @@
     <meta charset="UTF-8" />
```

Comment:
[OPTIONAL] I think it would be better if you can change this file name from main.html to index.html. Browsers automatically find the index.html file and show it as the main page of the app.